sdk/python/lib/pulumi/output.py

```diff
@@ -341,3 +357,21 @@ def concat(*args: List[Input[str]]) -> 'Output[str]':
 
         transformed_items = [Output.from_input(v) for v in args]
         return Output.all(*transformed_items).apply("".join)
+
+
+@known_types.unknown
```

Comment:
what does `@known_types.unknown` do?

Reply:
It ensures that the code in lib/pulumi/runtime and lib/pulumi agree on which class object refers to `Unknown`. See https://github.com/pulumi/pulumi/blob/master/sdk/python/lib/pulumi/runtime/known_types.py#L15-L31 for details.